.github/workflows/jib-cli-release.yml

```diff
@@ -44,13 +56,22 @@ jobs:
           release_name: jib-cli v${{ github.event.inputs.release_version }}
           draft: true
 
-      - name: Upload Jib CLI binaries
+      - name: Upload Jib CLI
+        uses: actions/upload-release-asset@v1
```

Comment:
Unfortunately, to [upload multiple assets](https://github.com/actions/upload-release-asset/issues/28#issuecomment-613045499), we need to add multiple steps.